app/src/main/java/org/techtown/soptseminar/FollowerFragment.kt

```diff
@@ -23,60 +29,99 @@ class FollowerFragment : Fragment() {
         container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
-        _binding = FragmentFollwerBinding.inflate(inflater, container, false)
-        initAdapter()
+        // 된다
+        homeActivity = activity as HomeActivity
+        // DataBinding 적용??
+        _binding = DataBindingUtil.inflate(inflater, R.layout.fragment_follwer, container, false)
+        if (!homeActivity.userData.isNullOrBlank()) {
+            followerAdapter = FollowerAdapter()
+            binding.rvFollower.adapter = followerAdapter
+            Log.d("UserData:", homeActivity.userData)
+            initUserInfoNetwork(homeActivity.userData)
+        }
```

Comment:
```suggestion
        val userData = (requireActivity() as? HomeActivity)?.userData
        // DataBinding 적용??
        _binding = DataBindingUtil.inflate(inflater, R.layout.fragment_follwer, container, false)
        if (!userData.isNullOrBlank()) {
            followerAdapter = FollowerAdapter()
            binding.rvFollower.adapter = followerAdapter
            Log.d("UserData:", userData)
            initUserInfoNetwork(userData)
        }
```
매번 userData에 접근하기 위해 `homeActivity.userData` 형식으로 사용하고 있어 이를 userData라는 변수에 할당해주면 좋을 것 같다는 생각이 들었습니다. :-)
그리고 `activity(getActivity())`와 `requireActivity()`는 둘 다 현재 Fragment가 붙어있는 Activity를 반환하지만, Exception이 발생할 경우 `requireActivity()`가 조금 더 구체적인 예외를 반환한다고 하네요.
마지막으로 `activity as HomeActivity`인 경우인데, 만일 `activity`로 얻은 Activity가 HomeActivity가 아닐 경우에는 문제가 생길 수 있습니다. Fragment는 여러 Activity에서 재사용될 수 있기 때문입니다. 그래서 `as` 대신 `as?`를 사용하면, 타입 캐스팅이 가능할 경우에만 타입 캐스팅을 해줌으로써 예외를 조금이라도 더 피할 수 있게 됩니다.

Reply:
좋은 가르침 감사합니다 👍  코드리뷰 반영했습니다 :D